_config.yml

```diff
@@ -853,6 +853,16 @@ pdf:
     # CDNJS, provided by cloudflare, maybe the best CDN, but not works in China
     #cdn: //cdnjs.cloudflare.com/ajax/libs/pdfobject/2.1.1/pdfobject.min.js
 
+# Mermaid tag
+mermaid:
+  enable: false
+  # Available themes: default | dark | forest | neutral
+  theme: forest
+  # Use 8.0.0 as default, jsdelivr as default CDN, works everywhere even in China
+  cdn: //cdn.jsdelivr.net/npm/mermaid@8.0.0/dist/mermaid.min.js
```

Comment:
Here u can just `@8`, not `@8.0.0`. This will mean latest major version will be loaded. See here for more information: https://www.jsdelivr.com

Reply:
Thanks, I’ll update